devel/libtapi/Portfile

```diff
@@ -99,26 +114,21 @@ if {${os.platform} eq "darwin" && ${os.major} < 11} {
     configure.cxxflags-append -U__STRICT_ANSI__
 }
 
-# add support for PPC architectures
-patchfiles-append       patch-0004-tapi-add-PPC-support.diff
-
-platform darwin {
-    if {${build_arch} in "ppc ppc64"} {
-        # fix a bug in Lexer.cpp, see https://systems.nic.uoregon.edu/internal-wiki/index.php?title=Tau_clang_plugin
-        patchfiles-append  patch-lexer.diff
-    }
-}
-
 # add a missing strnlen definition if needed
 patchfiles-append       patch-0006-strnlen.diff
 
 # build manpages for tapi
 patchfiles-append       patch-0007-build-manpages-for-tapi.diff
 
+# don't try to call pthread_setname_np on 10.5
+patchfiles-append       optional-pthread_setname_np.diff
+
 configure.post_args     ${worksrcpath}/src/llvm
 
 # needed for libtapi to find some required files
-configure.args-append   -DCMAKE_CXX_FLAGS="-I ${worksrcpath}/src/llvm/projects/clang/include -I ${workpath}/build/projects/clang/include"
+configure.cxxflags-append \
```

Comment:
Any reason why you've changed this line?

Reply:
`configure.cxx_stdlib libc++` adds to `cxxflags` `-stdlib` flag, which should be preserved. Old way dismiss it.

Reply:
Oh didn't notice that